node/tessel-export.js

```diff
@@ -952,6 +957,251 @@ Tessel.LED.prototype.read = function(callback) {
   });
 };
 
+Tessel.Wifi = function() {
+  var state = {
+    settings: {},
+    connected: false,
+    busy: false
```

Comment:
It's accessed through the `isBusy` property (changed from a method originally), https://github.com/tessel/t1-docs/blob/master/wifi.md#api-wifi-isBusy. `isBusy` is true while connecting wifi, resetting, disabling, and enabling, and false once the callback is called. Could be useful, but I was mostly trying to match the original spec.